broker/src/main/java/io/camunda/zeebe/broker/system/partitions/impl/steps/ZeebeDbPartitionTransitionStep.java

```diff
@@ -81,12 +81,11 @@ private void recoverDb(
             transitionFuture.completeExceptionally(error);
           } else {
             try {
-              context.getStateController().verifyDb();
               final var zeebeDb = context.getStateController().openDb();
               context.setZeebeDb(zeebeDb);
               transitionFuture.complete(null);
             } catch (final Exception e) {
-              Loggers.SYSTEM_LOGGER.error("Failed to recover from snapshot", e);
+              Loggers.SYSTEM_LOGGER.error("Failed to recover from snapshot.", e);
```

Comment:
:thought_balloon: Do you think it makes sense to add the snapshot to the output? :)

Reply:
We do not know the snapshot at this point.